packages/nomad/orchestrator.hcl

```diff
@@ -0,0 +1,80 @@
+variable "gcp_zone" {
+  type    = string
+}
+
+variable "port" {
+  type    = number
+  default = 5008
+}
+
+variable "consul_token" {
+  type    = string
+  default = ""
+}
+
+variable "logs_proxy_address" {
+  type    = string
+  default = ""
+}
+
+variable "otel_tracing_print" {
+  type    = string
+  default = ""
+}
+
+variable "environment" {
+  type    = string
+  default = ""
+}
+
+
+job "orchestrator" {
+  datacenters = [var.gcp_zone]
+
+  priority = 85
+
+  group "client-orchestrator" {
+    network {
+      port "orchestrator" {
+        static = var.port
+      }
+    }
+
+    service {
+      name = "orchestrator"
+      port = var.port
+
+      check {
+        type         = "grpc"
+        name         = "health"
+        interval     = "20s"
+        timeout      = "5s"
+        grpc_use_tls = false
+        port         = var.port
+      }
+    }
+
+    task "start" {
+      driver = "raw_exec"
+
+      resources {
```

Comment:
@jakubno Let's check if the raw exec resources limit the spawned FC resources (should not) and in case they do make it unlimited (https://developer.hashicorp.com/nomad/docs/drivers/raw_exec#resource-isolation).

Reply:
There's no resource (cpu/ram) isolation for raw exec